src/squid.md

```diff
@@ -0,0 +1,243 @@
+
+# NAME
+
+**squid** - HTTP web proxy caching server
+
+# SYNOPSIS
+
+**squid** [-dsCFNRSXY] [-\-foreground] [-l facility] [-f file] [-[au] port] [-k signal] [-n service]
```

Comment:
Dropped the '-V' option which is supported by the `getopts(2)` in `src/main.cc` (for Windows native build only) but does not have a switch case to handle it on command line parsing.
TODO remove the useless 'V' option support from Squid code.